### ISSUE TITLE
Latest expo powered project is unable to build ios and android

### ISSUE THREAD

**Issue by pycraft114** [infinitered/ignite#1899]
What's going on?

--- Latest expo powered project is unable to build ios and android

error The resource `/Users/chanwoopark/projects/PizzaApp/index.js` was not found.
Error: The resource `/Users/chanwoopark/projects/PizzaApp/index.js` was not found.
    at /Users/chanwoopark/projects/PizzaApp/node_modules/metro/src/IncrementalBundler.js:457:26
    at gotStat (fs.js:1936:21)
    at FSReqCallback.oncomplete (fs.js:191:21)
    
    index.js` was not found. error occurs for both android and ios

Steps to reproduce

1. npx ignite-cli new PizzaApp --expo
2. yarn  build-ios

---

`npx ignite-cli doctor` results:

System
  platform           darwin
  arch               x64
  cpu                8 cores      Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
  directory          PizzaApp     /Users/chanwoopark/projects/PizzaApp

JavaScript (and globally-installed packages)
  node               14.18.3      /Users/chanwoopark/.nvm/versions/node/v14.18.3/bin/node
  npm                6.14.15      /Users/chanwoopark/.nvm/versions/node/v14.18.3/bin/npm
    firebase-tools   10.1.0
    npm              6.14.15
  yarn               1.22.4       /usr/local/bin/yarn
  pnpm               -            not installed

Ignite
  ignite-cli         7.10.5       /Users/chanwoopark/.npm/_npx/93927/bin/ignite
  ignite src         build        /Users/chanwoopark/.npm/_npx/93927/lib/node_modules/ignite-cli/build

Android
  java               1.8.0_211    /usr/bin/java
  android home       -            undefined

iOS
  xcode              13.2.1
  cocoapods          1.11.2       /usr/local/bin/pod

**Comment by robinheinze** [comment]
@pycraft114 What happens when you run `yarn start` instead of `yarn build-ios`?

**Comment by sergioisidoro** [comment]
I just spent a few hours trying to figure out why I was getting these errors for iOS, but things were working in web:

```
iOS Bundling failed 482ms
While trying to resolve module `native-base` from file `DIR/app/app.tsx`, the package `DIR/node_modules/native-base/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`DIR/node_modules/native-base/dist/bundle.web.js`. Indeed, none of these files exist:

  * DIR/node_modules/native-base/dist/bundle.web.js(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)
  * DIR/node_modules/native-base/dist/bundle.web.js/index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)

```

Deleting metro config solved it. Maybe it's related?
https://github.com/infinitered/ignite/pull/1847

**Comment by jamonholmgren** [comment]
@sergioisidoro Probably related to #1904 -- fix incoming in #1905.

**Comment by jamonholmgren** [comment]
For this specific issue, you should use `yarn start`, not `yarn build-ios`. We should remove those commands for Expo projects.
